src/components/TodoItem/createTodoItem.js

```diff
@@ -0,0 +1,8 @@
+function createTodoItem({ itemName, itemStatus } = { itemName: '', itemStatus: false }, index = 0) {
```

Comment:
For readability, your code lines shouldn't have more than 80 characters long, if this is the case, consider breaking lines.